embc-app/ClientApp/src/app/registration-maker/registration-maker.component.ts

```diff
@@ -718,7 +718,12 @@ export class RegistrationMakerComponent implements OnInit {
     const s = value as string;
     return s ? s.trim() : null;
   }
-
+  cancel() {
+    // clear the loaded record if available
+    this.uniqueKeyService.clearKey();
+    // navigate back home
+    this.router.navigate([`/${this.path}/registrations`]);
```

Comment:
This is a different route than you removed in the HTML routerLink attribute... on purpose?

Reply:
Fixed